dlk/python/dlk/templates/src/func/pad.cpp

```diff
@@ -0,0 +1,38 @@
+#include "global.h"
```

Comment:
Please write License

Reply:
fixed the license with https://github.com/blue-oil/blueoil/pull/65/commits/a7d5a5b5c0b99405dda466ecbfd0ca2d5dba56ee